02 - Memtables, Joins, and Selectors.ipynb

```diff
@@ -463,7 +463,7 @@
    "outputs": [],
```

Comment:
<p>not sure why this is here but you already have this above <code>t2 = ibis.memtable(pat)</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis-tutorial/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Oops!</p>